max_sub_array/max_sub_array.py

```diff
@@ -0,0 +1,13 @@
+def max_sub_array_sum(a, size):
+    max_so_far = -9999999 - 1
+    max_ending_here = 0
+    for i in range(0, size):
+        max_ending_here = max_ending_here + a[i]
+        if (max_so_far < max_ending_here):
+            max_so_far = max_ending_here
+        if max_ending_here < 0:
+            max_ending_here = 0
+    return max_so_far
+# Driver function to check the above function  
+a = [-2, -3, 4, -1, -2, 1, 5, -3] 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp7_1_ei92/max_sub_array/max_sub_array.py
+++ b/tmp/tmp7_1_ei92/max_sub_array/max_sub_array.py
@@ -9,5 +9,5 @@
             max_ending_here = 0
     return max_so_far
 # Driver function to check the above function  
-a = [-2, -3, 4, -1, -2, 1, 5, -3] 
+a = [-2, -3, 4, -1, -2, 1, 5, -3]
 print("Maximum contiguous sum is", max_sub_array_sum(a, len(a)))
```